домашнее задание к уроку 8

```diff
@@ -0,0 +1,26 @@
+#домашнее задание №1
+
+import os.path
+
+project_path = 'my_project'
+paths = ['settings', 'mainapp', 'admiapp', 'authapp']
+for d in paths:
+    os.makedirs(os.path.join(os.curdir, project_path, d))
+    
+    
+    #домашнее задание №3
+
+import os
+
+SRC_FOLDER = 'me_project'
+DST_FOLDER = 'templates'
+
+dirs = os.walk(SRC_FOLDER)
+for x, y, _ in dirs:
+    for i in y:
+        newpath = os.path.join(os.curdir, DST_FOLDER, os.sep.join(x.split(os.sep)[1:]),i)
+        print(x.split(os.sep))
+        os.makedirs(newpath)
+        
+домашнее задание №4
+
```

Comment:
import os
import json

size_dict = {}
for root, dirs, files in os.walk('my_project'):
    for file in files:
        key_dict = os.stat(os.path.join(root, file)).st_size // 10 * 10 + 10
        f_ext = file.rsplit('.')[-1]
        if key_dict in size_dict:
            size_dict[key_dict][1].append(f_ext)
            size_dict[key_dicct] = (size_dict[key_dict][0] + 1, list(set(size_dict[key_dict][1])))
        else:
            size_dict.setdefault(key_dict,(1, [f_ext]))

res_d = {k: size_dict[k] for k in sorted(size_dict.keys())}
print(res_d)
with open('result.json', 'w') as f:
    json.dump(res_d, f)

with open('result.json', 'r') as f:
    print(json.load(f))